Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.py

```diff
@@ -306,22 +308,183 @@ def create_or_update_saved_search_command(client, args):
+        - Sends a 'PUT' request to create the search job using the provided parameters.
+        - Handles exceptions related to job existence.
+        - Returns a PollResult indicating to continue polling for job status.
+
+    - Subsequent Runs:
+        - Checks the status of the previously created search job.
+        - If the job is successful ('Succeeded'), it returns a PollResult to stop polling and provides
+          a readable output for success.
+        - If the job is still in progress, it returns a PollResult to continue polling.
+
+    The polling interval and timeout are determined based on the command arguments.
+    """
+    table_name: str = args['table_name']
+    if not table_name.endswith(TABLE_NAME_SUFFIX):
+        return_warning(f"The table_name should end with '{TABLE_NAME_SUFFIX}' suffix", exit=True)
+
+    if argToBoolean(args["first_run"]):
+        if start_search_time_datetime := arg_to_datetime(args.get('start_search_time', '1 day ago'), "start_search_time"):
+            start_search_time_iso = start_search_time_datetime.isoformat()
+        if end_search_time_datetime := arg_to_datetime(args.get('end_search_time', '1 day ago'), "end_search_time"):
+            end_search_time_iso = end_search_time_datetime.isoformat()
+        url_suffix = f"/tables/{table_name}"
+        data = {
+            "properties": {
+                "searchResults": {
+                    "query": args['query'],
+                    "limit": arg_to_number(args.get('limit')),
+                    "startSearchTime": start_search_time_iso,
+                    "endSearchTime": end_search_time_iso
+                }
+            }
+        }
+        try:
+            client.http_request(
+                'PUT',
+                url_suffix,
+                resource=AZURE_MANAGEMENT_RESOURCE,
+                data=data
+            )  # the response contain only the status code [202]
+        except Exception as e:
+            if "[InvalidParameter 400] This operation is not permitted as properties.searchResult is immutable." in e.args[0]:
+                return_warning(f"Search job {table_name} already exists - please choose another name", exit=True)
+            raise e
+        args["first_run"] = False
+        return PollResult(
+            response=None,
+            args_for_next_run=args,
+            continue_to_poll=True,
+            partial_result=CommandResults(
+                readable_output=(f"The command was sent successfully, to check the status of the command "
+                                 "please run the azure-log-analytics-get-search-job command "
+                                 "if the status of the provisioningState is Succeeded "
+                                 "you can run query on the table, using the azure-log-analytics-execute-query command "
+                                 f"with the query argument query={table_name}."),
```

Comment:
Why do we need this message?
I think it will be better to just use "The command was sent successfully, A temporary table is being created.
You check the status of the command by running !azure-log-analytics-get-search-job command or wait."

and then in the last response write:
"The {table_name} table created successfully.  In order to run queries on it, run `!azure-log-analytics-execute-query query={table_name}`.

Reply:
done